scripts/scripts.js

```diff
@@ -71,14 +71,48 @@ export function createTag(tag, attributes, html) {
   return el;
 }
 
+export function createModal(modalTitle, modalBodyInnerContent, modalContentClass, modalBodyClass,
+  modalId, modalContainerClass, buttonsConfig) {
+  let modalBodyInnerContentHtml = '';
+  if (modalBodyInnerContent !== '') {
+    modalBodyInnerContentHtml = modalBodyInnerContent.outerHTML;
+  }
+  const modalHeaderDiv = createTag('div', { class: 'modal-header' }, '');
```

Comment:
I saw `createTag` is used often in the code. I think we should avoid using it. Or be aware of it's risks. For the caller it is not obvious that the 3rd parameter poses a risk for XSS. I see many cases where strings are passed which don't seem to be sanitized. And there is often no need to parse these as HTML. 

instead dom-helpers.js can be used which is safe, or the content can be set safely with `el.textContent`

Reply:
Sure I will refactor this and use dom-helpers.js